src/tests/unit/tests/DetailsView/components/tab-stops/tab-stops-choice-group.test.tsx

```diff
@@ -36,54 +42,51 @@ describe('TabStopsChoiceGroup', () => {
     });
 
     test('render with unknown status', () => {
-        const testSubject = shallow(<TabStopsChoiceGroup {...props} />);
-        expect(testSubject.getElement()).toMatchSnapshot();
+        const renderResult = render(<TabStopsChoiceGroup {...props} />);
+        expect(renderResult.asFragment()).toMatchSnapshot();
+        expectMockedComponentPropsToMatchSnapshots([ChoiceGroup]);
     });
 
     test('render with unknown status (does not show undo button)', () => {
-        const testSubject = mount(<TabStopsChoiceGroup {...props} />);
-        expect(testSubject.find(IconButton).exists()).toBeFalsy();
+        const renderResult = render(<TabStopsChoiceGroup {...props} />);
+        expect(renderResult.queryByRole('button')).not.toBeTruthy();
     });
 
     test('render with fail status', () => {
         props.status = TabStopRequirementStatuses.fail;
-        const testSubject = shallow(<TabStopsChoiceGroup {...props} />);
-        expect(testSubject.getElement()).toMatchSnapshot();
+        const renderResult = render(<TabStopsChoiceGroup {...props} />);
+        expect(renderResult.asFragment()).toMatchSnapshot();
+        expectMockedComponentPropsToMatchSnapshots([ChoiceGroup, IconButton]);
     });
 
     test('render with pass status', () => {
         props.status = TabStopRequirementStatuses.pass;
-        const testSubject = shallow(<TabStopsChoiceGroup {...props} />);
-        expect(testSubject.getElement()).toMatchSnapshot();
+        const renderResult = render(<TabStopsChoiceGroup {...props} />);
+        expect(renderResult.asFragment()).toMatchSnapshot();
+        expectMockedComponentPropsToMatchSnapshots([ChoiceGroup]);
     });
 
-    test('verify component is correctly used with undo', () => {
+    test('verify component is correctly used with undo', async () => {
         props.status = TabStopRequirementStatuses.pass;
-        const testSubject = mount(<TabStopsChoiceGroup {...props} />);
-        const choiceGroupMock = Mock.ofType<IChoiceGroup>();
-        const eventStub = {} as React.MouseEvent<HTMLElement>;
-
-        const setComponentRef = testSubject.find(ChoiceGroup).prop('componentRef') as any;
-        const undoLinkOnClicked = testSubject.find(IconButton).prop('onClick');
+        useOriginalReactElements('@fluentui/react', ['ChoiceGroup', 'IconButton']);
+        const renderResult = render(<TabStopsChoiceGroup {...props} />);
 
-        setComponentRef(choiceGroupMock.object);
-        undoLinkOnClicked(eventStub);
+        const undoButton = renderResult.getAllByRole('button');
+        const radioButton = renderResult.getAllByRole('radio');
 
-        choiceGroupMock.verify(m => m.focus(), Times.once());
-        onUndoClickedMock.verify(m => m(eventStub), Times.once());
+        expect(undoButton).not.toBeNull();
+        fireEvent.click(undoButton[0]);
+        expect(radioButton[0]).toHaveFocus();
```

Comment:
in assertions do we have toHaveFocus(), i dont see this method in jest
https://jestjs.io/docs/expect